platform/macos/src/MGLMapView.mm

```diff
@@ -1198,7 +1198,7 @@ - (void)setCamera:(MGLMapCamera *)camera withDuration:(NSTimeInterval)duration a
         };
     }
     
-    if ([self.camera isEqualToMapCamera:camera]) {
+    if ([self.camera isEqualToMapCamera:camera] && NSEdgeInsetsEqual(edgePadding, NSEdgeInsetsZero)) {
```

Comment:
edgePadding specified here is self.contentInsets when specified in L.1181.
The documentation doesn't state that this is additional padding, like in https://github.com/mapbox/mapbox-gl-native/issues/12818#issuecomment-496836719, additional to self.contentInsets.
I believe that we need to check NSEdgeInsetsEqual(edgePadding, self.contentInsets)) here instead.

Reply:
Oof, thanks, somehow I missed that the macOS implementation is also affected by #12818.